pkg/controller/acicontainersoperator.go

```diff
@@ -329,6 +350,35 @@ func NewAciContainersOperator(
 			}
 		},
 	})
+
+	config_map_informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			key, err := cache.MetaNamespaceKeyFunc(obj)
+			log.Debug("The ConfigMap key: ", key)
```

Comment:
Assuming this event does not happen often, we should probably log this at Info level. Also, can you make the log a little more descriptive. One should not have to go back and look in the code to understand the context of the log statement.